reports/qc_exploration/qc_exploration.Rmd

```diff
@@ -0,0 +1,284 @@
+```
+
+```{r list_contents2}
+obj_list <- readRDS(glue("{data_dir}/{date}_list_contents.rds"))
+count(obj_list, ftype) %>% knitr::kable(caption = glue("Count of file types in {bucket}."))
+gvcf <-
+  obj_list %>%
+  dplyr::filter(ftype == "GVCF") %>%
+  dplyr::mutate(basename = basename(name),
+                tobid = sub(".g.vcf.gz", "", basename),
+                batch = sub("batch(.)", "\\1", basename(dirname(name)))) %>%
+  dplyr::select(tobid, fullname = name, batch, size, basename)
+```
+
+- Total of **`r nrow(obj_list)`** files, consisting of **`r nrow(gvcf)`**
+  GVCF files:
+
+```{r gvcf_list}
+gvcf_tab <- gvcf %>%
+  dplyr::mutate(size = as.character(size),
+                n = dplyr::row_number()) %>%
+  dplyr::select(n, fullname, batch, size) %>%
+  reactable::reactable(
+    pagination = FALSE, highlight = TRUE, height = 500, searchable = TRUE,
+    filterable = TRUE, bordered = TRUE, fullWidth = FALSE,
+    columns = list(
+      n = colDef(minWidth = 60),
+      fullname = colDef(minWidth = 420),
+      batch = colDef(minWidth = 60),
+      size = colDef(minWidth = 80)
+    )
+  )
+
+htmlwidgets::prependContent(
+  gvcf_tab,
+  htmltools::h2(class = "title", glue("GVCF files in {bucket} (Total: {nrow(gvcf)})")))
+```
+
+```{r gvcf_file_sizes}
+theme_set(theme_bw())
+
+gvcf %>%
+  summarise(
+    min = fs::as_fs_bytes(min(size)),
+    max = fs::as_fs_bytes(max(size)),
+    q1 = fs::as_fs_bytes(quantile(size, 0.25)),
+    median = fs::as_fs_bytes(median(size)),
+    q3 = fs::as_fs_bytes(quantile(size, 0.75)),
+    total = fs::as_fs_bytes(sum(size)),
+    .groups = "drop") %>%
+  tidyr::pivot_longer(cols = dplyr::everything()) %>%
+  knitr::kable(caption = glue("Size metrics for {nrow(gvcf)} GVCF files in {bucket}."))
+
+# jitter object for proper labels
+jitter_pos1 <- ggplot2::position_jitter(width = 0.10, height = 0, seed = 321)
+
+gvcf %>%
+  dplyr::mutate(
+    outlier = dplyr::if_else(is_outlier(size, 3),
+                             glue::glue("{tobid} ({size})"), NA_character_)) %>%
+  ggplot(aes(x = batch, y = size)) +
+  geom_violin(fill="lightblue") +
+  geom_point(position = jitter_pos1, shape = 21, color = "darkblue", fill = "red") +
+  ggrepel::geom_text_repel(position = jitter_pos1, aes(label = outlier),
+                           na.rm = TRUE, hjust = -0.3) +
+  scale_y_continuous(labels = scales::comma, breaks = scales::breaks_pretty(10)) +
+  theme(panel.grid.minor = element_blank()) +
+  ggtitle(glue::glue("Size (bytes) for {nrow(gvcf)} GVCF files in {bucket}."))
+```
+
+## QC metrics
+
+We have access to CSV files with summaries of sample QC metrics. Additional
+files have been provided containing:
+
+- the standard deviation of CRAM insert size, as calculated by Picard's
+  CollectInsertSizeMetrics;
+- the gender of each sample;
+- the age of each sample;
+
+
+```{r qc_plot_prep}
+qc1 <-
+  here(glue("{data_dir}/gvcf_batch1_R_210315_BINKAN1_1K1KDNA_M002.csv")) %>%
```

Comment:
Do I understand correctly that `gvcf_batch1_R_210315_BINKAN1_1K1KDNA_M002.csv` is identical to `gs://cpg-tob-wgs-main/gvcf/batch1/R_210315_BINKAN1_1K1KDNA_M002.csv`? Would be nice to point to it directly. Or even read `gs://cpg-tob-wgs-main/joint_vcf/v1/work/samples.csv`, which is generated by the joint-calling workflow using the input metadata.